src/access/for.ts

```diff
@@ -136,3 +138,42 @@ export async function getAccessFor(
   }
   return null;
 }
+
+/**
+ * Get an overview of what access is defined for a given set of actors: all Agents
+ * or all Groups.
+ *
+ * This function works with Solid Pods that implement either the Web Access
+ * Control spec or the Access Control Policies proposal, with some caveats:
+ *
+ * - If access to the given Resource has been set using anything other than the
+ *   functions in this module, it is possible that it has been set in a way that
+ *   prevents this function from reliably reading access, in which case it will
+ *   resolve to `null`.
+ * - It will only return access specified explicitly for the given actor (Agent
+ *   or Group). If additional restrictions are set up to apply to the given Agent
+ *   in a particular situation, those will not be reflected in the return value
+ *   of this function.
+ * - It will only return access specified explicitly for the given Resource.
+ *   In other words, if the Resource is a Container, the returned Access may not
+ *   apply to contained Resources.
+ * - If the current user does not have permission to view access for the given
+ *   Resource, this function will resolve to `null`.
+ *
+ * @param resourceUrl URL of the Resource you want to read the access for.
+ * @param actorType type of actor whose access is being read.
+ * @returns What access are set for the given resource, grouped by resp. Agent or Group.
+ */
+export async function getAccessForAll(
+  resourceUrl: UrlString,
+  actorType: Exclude<Actor, "public">,
+  options = internal_defaultFetchOptions
+): Promise<Record<UrlString, Access> | null> {
+  if (actorType === "agent") {
+    return await getAgentAccessAll(resourceUrl, options);
+  }
+  if (actorType === "group") {
+    return await getGroupAccessAll(resourceUrl, options);
+  }
+  return null;
```

Comment:
Given that the actor-specific function throws an error when an unknown actor type is passed, should this do so too?

If we keep it this when, then I think we should change this to:

```suggestion
  return null as never;
```

If I understand it correctly, that should mean that as soon as a new value (e.g. `authenticatedAgent`) is added to `Actor`, TypeScript will warn us that we haven't dealt with that in this function.

Reply:
Ah, the actor-specific function doesn't throw either. Then I'd just add the `as never` :)

Reply:
be2c4c98a0482558260202e988fb83a00cb8da11